src/cuga/backend/knowledge/routes.py

```
@@ -0,0 +1,487 @@
+    initializer = getattr(app_state, "initialize_knowledge_engine", None)
+    if not initializer:
+        raise HTTPException(status_code=503, detail="Knowledge engine initializer not available")
+
+    try:
+        from cuga.backend.knowledge.config import KnowledgeConfig
+
+        kb_config = KnowledgeConfig.from_settings(__import__("cuga.config", fromlist=["settings"]).settings)
+        kb_config.enabled = True  # Force enable regardless of settings file
+        await initializer(app_state, kb_config)
+        return {"status": "started"}
+    except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)